### ISSUE TITLE
Inconsistency when coloring cell with apply last cell color and open cell colors

### ISSUE THREAD

**Issue by ME7DYCgw8M** [aardappel/treesheets#662]
Reproducing:

# Case 1
1. Make cell (cell 1)
2. Make subcell (cell 2)
3. Color cell 2 with ALT+SHIFT+F9 (open cell colors) red
4. Color cell 3 (any other cell but not cell 1 or cell 2) with ALT+SHIFT+F9 (open cell colors) green
5. Color cell 1 with ALT+SHIFT+C (apply last cell color)
6. Cell 2 is also colored with cell 3 color green

**TLDR: "apply last cell color" colors also subcells**

---

# Case 2
1. Make cell (cell 1)
2. Make subcell (cell 2)
3. Color cell 2 with ALT+SHIFT+F9 (open cell colors) with red
5. Color cell 1 with ALT+SHIFT+F9 (open cell colors) with green
6. Cell 2 stays red

**TLDR: "open cell colors" does not color subcells**

---

In case 1 subcell is colored with parent cell color. In case 2 subcell color is unaffected by cell 1 color change.

If this intentional, so be it.
But I think both cases should leave subcell coloring red.

Ps. Treesheets is awesome!
